src/legacy/core_plugins/kbn_vislib_vis_types/public/histogram.js

```diff
@@ -101,6 +101,13 @@ export default function PointSeriesVisType(Private) {
         addTimeMarker: false,
         labels: {
           show: false,
+        },
+        thresholdLine: {
+          show: false,
+          value: 10,
+          width: 1,
+          style: 'full',
+          color: '#00B3A4'
```

Comment:
@cchaos Do we have a way to import this color (`$euiColorVis0`) from the SASS somehow into the JS code? I unfortunately wasn't able to find something in the EUI docs around that?

Reply:
Yes, we actually provide a [palette service](https://elastic.github.io/eui/#/utilities/color-palettes) where you can do something like:

```js
import { palettes } from '../../../../src/services';

const color: palettes.euiPaletteColorBlind.colors[0];
```

Reply:
However, looking at the screenshot, that's not going to visible over top of the bars of the same color. I'd suggest using the last color in that palette instead. 

```js
const color = palettes.euiPaletteColorBlind.colors[9];
```